src/Compilers/CSharp/Portable/Binder/Binder_Conversions.cs

```diff
@@ -2788,7 +2788,7 @@ static ErrorCode getRefMismatchErrorCode(TypeKind type)
                 return true;
             }
 
-            var sourceMethod = selectedMethod as SourceOrdinaryMethodSymbol;
+            var sourceMethod = selectedMethod.OriginalDefinition as SourceOrdinaryMethodSymbol;
             if (sourceMethod is object && sourceMethod.IsPartialWithoutImplementation)
```

Comment:
>IsPartialWithoutImplementation [](http://example.com/codeflow?start=55&length=30)

Consider checking other use sites of this API

Reply:
There was only one other use of `IsPartialWithoutImplementation`, in `CallsAreOmitted()`.